projects/fsastorefrontlib/src/core/icon-config/icon-config.ts

```diff
@@ -0,0 +1,26 @@
+import { fontawesomeIconConfig, IconConfig } from '@spartacus/storefront';
+
+export const iconConfig: IconConfig = {
+  ...fontawesomeIconConfig,
+  icon: {
+    symbols: {
+      PROPERTY: 'fs-icon icon-house',
+      AUTO: 'fs-icon icon-auto',
+      PERSON: 'fs-icon icon-heart',
+      TRIP: 'fs-icon icon-plane',
+      EVENT: 'fs-icon icon-event',
+      PHONE: 'fas fa-phone',
+      AGENT: 'fas fa-headset',
+    },
+  },
+};
+
+export enum ICON_TYPE {
```

Comment:
Perhaps rename this to FS_ICON_TYPE to avoid confusion with Spartacus ICON_TYPE?